modules/virt-descheduler-installing.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/advanced_vm_management/virt-enabling-descheduler-evictions.adoc
+
+:_content-type: PROCEDURE
+[id="virt-descheduler-installing_{context}"]
+= Installing the descheduler to use on a VM
+
+The descheduler is not available by default. To enable the descheduler, you must install the Kube Descheduler Operator from OperatorHub and enable one or more descheduler profiles.
+
+Use the Technology Preview `DevPreviewLongLifecycle` profile to enable the descheduler on a virtual machine. To ensure proper scheduling, create VMs with CPU and memory requests for the expected load.
```

Comment:
I think the `DevPreviewLongLifecycle` profile description can be moved to a separate reference module. That way you can reuse the OCP descheduler installation module (`nodes-descheduler-installing.adoc`) without having to create a new one for CNV. 
You'd have to use [conditionals ](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#conditionals-for-assemblies)so the CNV-specific content in the last couple of steps gets published for this assembly. We can discuss this offline if you have any questions.